string robot_namespace
AABB bbox